src/index.css

```
@@ -1,68 +1,51 @@
 
-h1 {
-  font-size: 3.2em;
-  line-height: 1.1;
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.11.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)